components/dialog/stories/template.js

```diff
@@ -13,9 +14,8 @@ import "../index.css";
 
 export const Template = ({
 	rootClass = "spectrum-Dialog",
-	isDismissable = true,
-	isFullScreen = false,
-	isFullScreenTakeover = false,
+	isDismissible = false,
```

Comment:
```suggestion
	isDismissible = false,
```
^ Was this a change in the design documentation? Is being dismissible no longer the default for a dialog?

Reply:
@castastrophe Being totally honest, I don't know what the default is. I haven't found that super clear, but if that's the default, I'll change it!! Where would I find that info?

I do know, that after we [reimplemented the buttons on the docs site](https://github.com/adobe/spectrum-css/pull/2861), where those dialogs didn't have anything, (no close button _or_ button group), I had added them to most of dialogs (obviously not the previously dismissible ones). I was probably referencing that page with my own changes 🤦‍♀️ Let me know if I need to change this back- I'm absolutely open to it.